### DISCUSSION TITLE
GitHub Action trigger new check run

### DISCUSSION THREAD

**Discussion by podviaznikov** [community/community#26132] [Actions]
<p>I have the following use case:</p>
<p>I would like to implement linter that would run as part of action and report check run with annotations (report with warning with location information).</p>
<p>I don’t think it’s possible to do it as part of Action. I see that Action can’t make API calls that will trigger other GitHub Actions.</p>
<p>Is there any solution for this on the roadmap?</p>

**Comment by gimenete** [comment, marked as answer]
<p>Hi,</p>
<p>It is possible. In fact I made three actions that create check run annotations</p>
<p><a href="https://github.com/gimenete/rubocop-action" rel="nofollow noopener">https://github.com/gimenete/rubocop-action</a><br>
<a href="https://github.com/gimenete/eslint-action" rel="nofollow noopener">https://github.com/gimenete/eslint-action</a></p>
<p><a href="https://github.com/gimenete/github-jest-snapshots" rel="nofollow noopener">https://github.com/gimenete/github-jest-snapshots</a></p>
<p>The first one is implemented in Ruby and the other ones are implemented in Node.js.</p>
